buildlib/az-helpers.sh

```diff
@@ -163,3 +163,15 @@ check_release_build() {
 
     echo "##vso[task.setvariable variable=Launch;isOutput=true]${launch}"
 }
+
+#
+# Returns free random port in private port range.
+#
+function EPHEMERAL_PORT() {
+    while :
+    do
+            PORT="`shuf -i 49152-65535 -n 1`"
+            ss -lpn | grep -q ":$PORT " || break
```

Comment:
we cannot guarantee a port will remain unused..
better take It from EXECUTOR_NUMBER same formula that is used for other TCP tests